src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,12 @@
+@mixin media-query($breakpoint, $columns) {
+  @media (min-width: $breakpoint) {
+    .cards {
+      grid-template-columns: repeat($columns, $card-width);
+    }
+  }
+}
+
+@include media-query(0, 1);
+@include media-query(488px, 2);
+@include media-query(768px, 3);
+@include media-query(1024px, 4);
```

Comment:
Consider will be better if you move sizes to variables and use it here

Reply:
i.e. I need to create variables and use them? or just move those includes to a variable file?